Cargo.lock

```diff
@@ -8126,7 +8126,6 @@ dependencies = [
  "md-5",
  "once_cell",
  "percent-encoding",
- "prometheus",
```

Comment:
How could this happen??

Reply:
Oh, I understood this now. Since we have multiple dependencies on opendal. The one enables `promethus` is upgraded.